gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowStatusResource.java

```diff
@@ -155,48 +152,11 @@ private FlowStatus convertFlowStatus(org.apache.gobblin.service.monitoring.FlowS
     return new FlowStatus()
         .setId(new FlowStatusId().setFlowGroup(flowId.getFlowGroup()).setFlowName(flowId.getFlowName())
             .setFlowExecutionId(monitoringFlowStatus.getFlowExecutionId()))
-        .setExecutionStatistics(new FlowStatistics().setExecutionStartTime(flowStartTime)
+        .setExecutionStatistics(new FlowStatistics().setExecutionStartTime(monitoringFlowStatus.getFlowExecutionId())
```

Comment:
maybe create a method like DagManagerUtils.getFlowStartTime() for improved readability